common/src/main/java/no/difi/meldingsutveksling/config/IntegrasjonspunktProperties.java

```diff
@@ -22,6 +25,11 @@
 @ConfigurationProperties(prefix = "difi.move")
 public class IntegrasjonspunktProperties {
 
+    @PostConstruct
+    private void postConstruct() {
+        MDC.put(MoveLogMarkers.KEY_ORGANISATION_NUMBER, getOrg().getNumber());
+    }
+
     @Valid
     private Organization org;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)